src/CoreMedia/CMTime.cs

```diff
@@ -375,6 +376,7 @@ public static CMTime FromDictionary (NSDictionary dict)
 		}
 
 #if !XAMCORE_2_0
+		[NoWatch]
```

Comment:
`[NoWatch]` is not removing this code - i.e. this is not bindings
There's no point in having this code inside the platform assembly if obsoleted.
You need to use `#if !WATCH` in those cases